NavigationReactNative/src/ios/NavigationModule.m

```diff
@@ -35,13 +35,19 @@ - (dispatch_queue_t)methodQueue
         [navigationController popToViewController:navigationController.viewControllers[crumb] animated:true];
     }
     if (crumb > currentCrumb) {
-        NSMutableArray *controllers = [navigationController.viewControllers mutableCopy];
+        NSMutableArray *newControllers = [[NSMutableArray alloc] init];
```

Comment:
Keep the name `controllers` instead of `newControllers` so that it matches the [similar loop in the Android `NavigationModule`](https://github.com/grahammendick/navigation/blob/6e18b43e41479f7a795ef1ce29a97483e36ade63/NavigationReactNative/src/android/app/src/main/java/com/navigation/reactnative/NavigationModule.java#L102)